src/infra/auth/jwt.ts

```diff
@@ -47,6 +47,14 @@ export interface TokenUserPayload
 	authTime?: number;
 }
 
+const jwtValidFields = [
```

Comment:
We might later (during the bC cleanup) want to export this (so that consumers can augment it) and move the `_.pick` in the `createSessionToken()`.